ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/requirements.txt

```diff
@@ -2,3 +2,10 @@
 /opt/dataflow/prepackaged_df-1.0-py3-none-any.whl
 oci==2.74.0
 pandas==1.1.5
+
+setuptools~=49.6.0
+numpy~=1.19.5
+pyspark~=3.0.2
```

Comment:
pyspark must not be included as a dependency as per Data Flow documentation. this can cause issues

Reply:
I do not think it is a good idea to use the same requirements.txt to generate the archive file to run on DataFlow and so added the changes thinking this requirements.txt was for local run. But, thanks for sharing the information that this file will be used for generating the archive. I will remove the changes I added to this file.